client/btxn-service-rest-client/src/main/java/org/hawkular/btm/btxn/service/rest/client/BusinessTransactionServiceRESTClient.java

```diff
@@ -0,0 +1,288 @@
+/*
+ * Copyright 2015 Red Hat, Inc. and/or its affiliates
+ * and other contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.hawkular.btm.btxn.service.rest.client;
+
+import java.util.Base64;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import javax.ws.rs.client.Client;
+import javax.ws.rs.client.ClientBuilder;
+import javax.ws.rs.client.Entity;
+import javax.ws.rs.client.Invocation.Builder;
+import javax.ws.rs.client.WebTarget;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.core.Response.Status;
+
+import org.hawkular.btm.api.log.MsgLogger;
+import org.hawkular.btm.api.model.btxn.BusinessTransaction;
+import org.hawkular.btm.api.model.btxn.CorrelationIdentifier;
+import org.hawkular.btm.api.services.BusinessTransactionCriteria;
+import org.hawkular.btm.api.services.BusinessTransactionService;
+import org.jboss.logging.Logger;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+/**
+ * This class provides the REST client implementation for the Business Transaction Service
+ * API.
+ *
+ * @author gbrown
+ */
+public class BusinessTransactionServiceRESTClient implements BusinessTransactionService {
+
+    private static final MsgLogger msgLog = MsgLogger.LOGGER;
+
+    private static final Logger log = Logger.getLogger(BusinessTransactionServiceRESTClient.class);
+
+    private static final TypeReference<java.util.List<BusinessTransaction>> BUSINESS_TXN_LIST =
+            new TypeReference<java.util.List<BusinessTransaction>>() {
+            };
+
+    private static final ObjectMapper mapper = new ObjectMapper();
+
+    private static final String HAWKULAR_TENANT = "Hawkular-Tenant";
```

Comment:
Accounts uses the "Hawkular-Persona" header. I think that Metrics uses "Hawkular-Tenant".